### PR TITLE
Adds more bronze tools

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Now you can craft bronze tools for epic power gamer .5 speed buff!

## Why It's Good For The Game

Bronze is legit rarely used out side of CWC cheap wall making. Lets give it some uses
## Changelog
:cl:
add: stuff n things
/:cl:

### CODE REVIEW

File: code/modules/research/designs/autolathe_desings/autolathe_designs_tools.dm [hunk 1]
```diff
@@ -20,6 +20,14 @@
 	build_path = /obj/item/crowbar
 	category = list("initial","Tools")
 
+/datum/design/crowbar
+	name = "Large Crowbar"
+	id = "large_crowbar"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 500)
+	build_path = /obj/item/crowbar/large
+	category = list("initial","Tools")
```
Comment (deathride58) [inline code/modules/research/designs/autolathe_desings/autolathe_designs_tools.dm]: I'm extremely apprehensive about this. I really don't want to see tool speeds, of all things, end up with powercreep as a result of a 0.5 toolspeed admin-only item getting added

**Comment by silicons** [comment]
okay for what reason should we have high speed tools available roundstart in an autolathe of all things.

File: code/modules/research/designs/autolathe_desings/autolathe_designs_tools.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/modules/research/designs/autolathe_desings/autolathe_designs_tools.dm]: Oh then ill remove it

**Review by silicons — APPROVED** [review]

**Comment by silicons** [comment]
it was changed to be crafting only and only with 0.95 toolspeed let's do this then

**Pull request merged**